### ISSUE TITLE
Question: How do you recommend disabling the default modeline?

### ISSUE THREAD

**Issue by jgarte** [rougier/nano-modeline#62]
Hi,

How do you recommend disabling the default modeline?

![image](https://github.com/rougier/nano-modeline/assets/47760695/85663feb-8e71-4b1d-b8b7-be9bd16eae08)

This is what is in my config:

![image](https://github.com/rougier/nano-modeline/assets/47760695/ffb9415d-f864-44e2-901e-512b6bedc37d)

**Comment by rougier** [comment]
With the same hooks, you can use `(setq mode-line-format nil)`

**Comment by jgarte** [comment]
Makes sense. Thanks for sharing that knowledge. Closing this issue.

**Comment by jgarte** [comment]
Should this be mentioned in the README somewhere?

The reason I am suggesting this is because following the configuration recommendations in the README I ended up with two mode lines as shown above. I then went wondering how to disable the default mode line and came here.

**Comment by rougier** [comment]
You're right actually, it couldm be mentioned in the README. Do you want to make a PR?

**Comment by jgarte** [comment]
Sure

**Comment by rougier** [comment]
I'm actually not sure so maybe it is safe to disable header or mode line depending on where the nano modeline is. Best would probably to have a custom settings telling nano-modeline to disable mode/header line when in use.

**Comment by jgarte** [comment]
Hi, thanks for the reply.

Can you elaborate on why it is not safe? I don't think I am understanding that part.
